sdk/containerregistry/container-registry/review/container-registry.api.md

```diff
@@ -35,19 +31,132 @@ export class ContainerRegistryUserCredential {
     }
 
 // @public
-export type ContentProperties = ChangeableAttributes;
+export class ContainerRepositoryClient {
+    constructor(endpointUrl: string, repository: string, credential: TokenCredential | ContainerRegistryUserCredential, options?: ContainerRegistryClientOptions);
+    delete(options?: DeleteOptions): Promise<DeleteRepositoryResult>;
+    deleteRegistryArtifact(digest: string, options?: DeleteRegistryArtifactOptions): Promise<RestResponse>;
+    deleteTag(tag: string, options?: DeleteTagOptions): Promise<RestResponse>;
+    endpoint: string;
+    getProperties(options?: GetPropertiesOptions): Promise<RepositoryProperties>;
+    getRegistryArtifactProperties(tagOrDigest: string, options?: GetRegistryArtifactPropertiesOptions): Promise<RegistryArtifactProperties>;
+    getTagProperties(tag: string, options?: GetTagPropertiesOptions): Promise<TagProperties>;
+    listRegistryArtifacts(options?: ListRegistryArtifactsOptions): PagedAsyncIterableIterator<RegistryArtifactProperties>;
+    listTags(options?: ListTagsOptions): PagedAsyncIterableIterator<TagProperties>;
+    registry: string;
+    repository: string;
+    setManifestProperties(digest: string, value: ContentProperties, options?: SetManifestPropertiesOptions): Promise<RestResponse>;
```

Comment:
Reading the docs for azure container registry, it sounds like each artifact has a manifest associated with it. Should this be renamed `setArtifactManifestProperties`?

https://docs.microsoft.com/en-us/azure/container-registry/container-registry-concepts#manifest

Reply:
A digest could reference a manifest list too.  @annelo-msft please correct me if I am wrong.